src/Features/VisualBasic/Portable/InitializeParameter/InitializeParameterHelpers.vb

```diff
@@ -15,9 +15,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.InitializeParameter
 
         Public Shared Function GetBlockOperation(functionDeclaration As SyntaxNode, semanticModel As SemanticModel, cancellationToken As CancellationToken) As IBlockOperation
             Dim operation = semanticModel.GetOperation(functionDeclaration, cancellationToken)
-            Return If(TypeOf operation Is IAnonymousFunctionOperation,
-                DirectCast(operation, IAnonymousFunctionOperation).Body,
-                DirectCast(operation, IBlockOperation))
+            If TypeOf operation Is IAnonymousFunctionOperation Then
+                Return DirectCast(operation, IAnonymousFunctionOperation).Body
+            ElseIf TypeOf operation Is IBlockOperation Then
+                Return DirectCast(operation, IBlockOperation)
```

Comment:
instead of typeof+directcast, you can do TryCast+null check.  but this is minor nit.  in practice this shouldn't be that impactful.

Reply:
You can even consider a switch statement based on `OperationKind` and then direct cast to corresponding operation.